MozillaFoundation/foundation.mozilla.org
# PNI - Articles nav link should be highlighted

# Description

This pull request modifies the path check to see if the target is within the current path. It was previously only checking if the target was equal to the current path, and hence only worked on /articles pages.

Link to sample test page:
Related PRs/issues: #11126

# To Test

Pulled the acceptance criteria Mavis provided in the issue:

1) Navigate to the /articles category page. Make sure the "Articles" nav link is highlighted.
2) Visit an individual article. Make sure that the "Articles" nav link is highlighted.

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP-296)
